### DISCUSSION TITLE
Does it work with obfuscated import table?

### DISCUSSION THREAD

**Discussion by blackxored** [vfsfitvnm/frida-il2cpp-bridge#128] [Q&A]
I tried two games with this, it seems to work on the one that does 0 obfuscation, but not on the other. Are there any APIs that need to be on UnityPlayer or GameAssembly? Also the game in question does rename one of these files.

EDIT: I just looked at some code, `GameAssembly.dll` is indeed hardcoded, so fixing that, will see what else I run into.

**Comment by vfsfitvnm** [comment]
Hi, I don't think there's much I can do if a company replaces symbol names, your solution is perfectly fine - just change the hardcoded string(s) in `./node_modules/frida-il2cpp-bridge/dist`.

However, if the whole library is obfuscated, you can see the solution proposed in #33

**Comment by vfsfitvnm** [comment]
I'm closing this as there was no update - please reopen if needed
